src/js/page.js

```diff
@@ -20,6 +20,7 @@ export default {
         if ( this.form.view.$.hasClass( 'pages' ) ) {
             const $allPages = this.form.view.$.find( ' .question:not([role="comment"]), .or-appearance-field-list' )
                 .add( '.or-repeat.or-appearance-field-list + .or-repeat-info' )
+                .add( '.or-group' )
```

Comment:
I think this turns all groups into a page. That's not right. I don't think this needs any change. There are no known bugs in how a form is divided into pages, so that should not changes. This feature is purely ToC related.